src/drawers/areaDrawer.ts

```diff
@@ -78,10 +78,13 @@ export module _Drawer {
       if (attrToProjector["fill"]) {
         this._areaSelection.attr("fill", attrToProjector["fill"]); // so colors don't animate
       }
-      step.animator.animate(this._areaSelection, attrToProjector);
+      if (attrToProjector["class"]) {
+        this._areaSelection.attr("class", attrToProjector["class"]);
+        this._areaSelection.classed(Area.AREA_CLASS, true);
+        delete attrToProjector["class"];
```

Comment:
My understanding is that I think a lot of the confusion seems to arise from a sort of grungy architecture that is in the Drawers.  I may or may not look into it to see if it can be cleaned up.